apps/wallet/src/generated/station/station.did

```diff
@@ -2770,24 +2784,136 @@ type InitAssetInput = record {
   metadata : vec AssetMetadata;
 };
 
-// The init configuration for the canister.
+// The input type for creating a user group when initializing the canister for the first time.
+type InitUserGroupInput = record {
+  // The id of the user group.
+  id : opt UUID;
+  // The name of the user group, must be unique.
+  name : text;
+};
+
+// The input type for adding identities to a user.
+type UserIdentityInput = record {
+  // The identity of the user.
+  identity : principal;
+};
+
+// The users to create when initializing the canister for the first time.
+type InitUserInput = record {
+  // The id of the user, if not provided a new UUID will be generated.
+  id : opt UUID;
+  // The name of the user.
+  name : text;
+  // The identities of the user.
+  identities : vec UserIdentityInput;
+  // The user groups to associate with the user (optional).
+  // If not provided it defaults to the `admin` group if default user groups are created,
+  // i.e., when the field `entries` in `SystemInit` is `null` or has the form of `WithDefaultPolicies`.
+  groups : opt vec UUID;
+  // The status of the user (e.g. `Active`).
+  //
+  // If not provided the default status is `Active` when there is at least
+  // one identity or `Inactive` otherwise.
```

Comment:
The status seems to be always `Active` by default (independently of identities). (I guess the comment is out-dated.)

Reply:
No no, the active/inactive logic seems to have never actually existed. Adding it, to the service.

Reply:
I think it's a good addition as an active user might be required to vote, when they can't authenticate themselves.

Reply:
The alternative is to reject this as invalid input (the advantage being a simpler semantics of the default value).

Reply:
There might be value in being able to create a user which doesn't have an identity yet, eg. when configuring a team station.